world.cpp

```diff
@@ -42,7 +42,12 @@ LogMapNumberResult_t load_map_number(S32 map_number, Coord_t* player_start, Worl
 
      // search through directory to find file starting with 3 digit map number
      DIR* d = opendir("content");
-     if(!d) return result;
+     if(!d){
+         printf("load_map_number: opendir() failed\n");
+         return result;
+     }else{
+         printf("load_map_number: opendir() success\n");
```

Comment:
Unless you have a good reason, I prefer not to have messages that just claim success (especially when they get printed everytime we run the game)

Reply:
Gotcha. Should have removed that before committing; it was a temporary way to check if it was working in some cases.